Functions/Statistics/xASL_stat_PrintStats.m

```diff
@@ -46,7 +46,7 @@
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % EXAMPLE: x = xASL_stat_PrintStats(x);
 % __________________________________
-% Copyright 2015-2020 ExploreASL
+% Copyright 2015-2024 ExploreASL
```

Comment:
Actually, I haven't really modified the I/O. I only fixed an error from previously.

Any propositions what to change and how?

Reply:
Yes, e.g., you changed a variable name that was just copied to a new variable so was useless. But I agree that doesn't have to be in the header. If you didn't change the functionality you don't need to change the header of course, I thought that you changed the functionality but you probably didn't (e.g., if you disabled something it would be nice to put it to the header for us to remember)

Reply:
Agree in general!